qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -53,34 +54,36 @@ def _get_cirq_quregs(signature: Signature, qm: InteropQubitManager):
     return ret
 
 
-@frozen
-class CirqGateAsBloq(GateWithRegisters):
+class CirqGateAsBloqBase(GateWithRegisters):
     """A Bloq wrapper around a `cirq.Gate`"""
 
-    gate: cirq.Gate
+    @property
+    @abc.abstractmethod
+    def cirq_gate(self) -> cirq.Gate:
```

Comment:
update class docstring .. this is no longer a wrapper, right? it's something else